model/atmosphere/diffusion/tests/diffusion_tests/test_diffusion.py

```diff
@@ -624,7 +625,8 @@ def test_run_diffusion_multiple_steps(
 
 @pytest.mark.datatest
 @pytest.mark.parametrize("experiment", [dt_utils.REGIONAL_EXPERIMENT])
-@pytest.mark.parametrize("linit, orchestration", [(True, [True, False])])
+@pytest.mark.parametrize("linit", [True, True])
```

Comment:
Typo (my fault, sorry):
```suggestion
@pytest.mark.parametrize("linit", [True])
```